tinygrad/tensor.py

```diff
@@ -133,9 +133,13 @@ def eye(dim, **kwargs): return Tensor([1], **kwargs).slice(((0,dim+1),)).reshape
 
   # TODO: below line, remove use of numpy here and make lazy
   # TODO: requires cumsum to remove numpy
+  
   @staticmethod
-  def arange(stop, start=0, step=1, **kwargs): return Tensor(np.arange(start=start, stop=stop, step=step, dtype=np.float32), **kwargs)
-
+  def cumsum(x: Iterable, **kwargs): return Tensor([i for i in itertools.accumulate(x)], **kwargs)
```

Comment:
No, why would this be on the Tensor class?!?

Reply:
I assumed it would be better to put it in helper.py module as a helper function. 
https://github.com/dee7ine/tinygrad/blob/a44c7e5504a25052f86c7e21bb0d5175c1e3ece5/tinygrad/helpers.py#L16-L19

One thing I couldn't find in the discussion is why numpy should be removed. Is it to remove any dependencies?

Reply:
I assumed it could be better to move it to helpers.py module. 
https://github.com/geohot/tinygrad/blob/a44c7e5504a25052f86c7e21bb0d5175c1e3ece5/tinygrad/helpers.py#L17-L19